limitador-server/src/main.rs

```diff
@@ -139,7 +140,8 @@ impl Limiter {
             .response_timeout(Duration::from_millis(cache_cfg.response_timeout));
 
         cached_redis_storage.build().await.unwrap_or_else(|err| {
-            eprintln!("Failed to connect to Redis at {redis_url}: {err}");
+            let redacted_redis_url = redacted_url(String::from(redis_url));
+            eprintln!("Failed to connect to Redis at {redacted_redis_url}: {err}");
```

Comment:
The down side is that if `err` ever contains that password, we'd "reintroduce" the bug. Or do we know that is guarantee provided by the crate?

Reply:
I added verification steps with a manual test. 

I would never expect the crate to return the url of the connection serialized in the error string, but who knows maybe one day. Today, according to my tests, it is not doing so. Would you suggest to apply some regex replace to the string before writing it to stderr?

Reply:
let me know how you would dealt with the error otherwise. Happy to consider it